app/api/games/route.tsx

```diff
@@ -1,22 +1,49 @@
-import prisma from "@/prisma/prisma";
-
-export async function POST(req: Request) {
-	const body = await req.json();
-
-	const result = await prisma.game.create({
-		data: {
-			...body,
-		},
-	});
-	if (!result)
-		return Response.json({
-			message: "error",
-			status: 500,
-		});
-	return Response.json({ message: "ok", status: 200, data: result });
+import prisma from "@/app/api/games/validation";
+
+export async function POST(request: Request) {
+  const body = await request.json();
+
+  try {
+    const result = await prisma.game.create({
+      data: {
+        ...body,
+      },
+    });
+
+    return new Response(JSON.stringify({ result }), { status: 201 });
+  } catch (e) {
+    return new Response(JSON.stringify({ message: "Validation error" }), {
+      status: 400,
+    });
+  }
 }
 
 export async function GET(req: Request) {
-	const result = await prisma.game.findMany();
-	return Response.json({ message: "ok", status: 200, data: result });
+  const result = await prisma.game.findMany();
+  return Response.json({ message: "ok", status: 200, data: result });
+}
+
+export async function DELETE(request: Request) {
+  const body = await request.json();
+  const idsToDelete = body.ids;
+
+  if (!Array.isArray(idsToDelete)) {
+    alert("Incorret data. Please check fields");
+  }
```

Comment:
There is no `alert` on server side. This is API, so in case of incorrect requests being made you should be responsing with [400](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/400)